makefile

```
@@ -0,0 +1,141 @@
+SHELL           := $(shell which bash)
+    
+NO_COLOR   .    := \033[0m
```

With the dot in the NO_COLOR variable 
```suggestion
NO_COLOR       := \033[0m
```
On bash:
```makefile:3: *** empty variable name.  Stop.```